docs/messaging/rabbitmq.md

```diff
@@ -20,6 +20,7 @@ In RabbitMQ, the destination is defined by an *exchange* and a *routing key*.
 <!-- semconv messaging.rabbitmq -->
 | Attribute  | Type | Description  | Examples  | Requirement Level |
 |---|---|---|---|---|
+| `messaging.rabbitmq.delivery_tag` | int | RabbitMQ delivery tag. | `123` | Conditionally Required: When messages are received or delivered. |
```

Comment:
```suggestion
| `messaging.rabbitmq.delivery_tag` | int | RabbitMQ delivery tag. | `123` | Conditionally Required: For consumer operations. |
```

Maybe you'll want to add a `settle` operation one day where you also want to have this tag?

Reply:
or just `when available`.

Make sure to modify yaml file and generate the markdown table with `docker run -v /mnt/c/repo/semantic-conventions/model:/source -v /mnt/c/repo/semantic-conventions/docs:/dst otel/semconvgen:0.22.0 -f /source markdown -md /dst`  (just change the path)